packages/schematics/core/lang.config.ts

```diff
@@ -20,6 +20,7 @@ export const LANGS_CONFIG = [
   { langs: ['el'], zorro: 'el_GR', delon: 'el_GR', fileName: 'el-GR.json' },
   { langs: ['ko'], zorro: 'ko_KR', delon: 'ko_KR', fileName: 'ko-KR.json' },
   { langs: ['hr'], zorro: 'hr_HR', delon: 'hr_HR', fileName: 'hr-HR.json' },
+  { langs: ['ja'], zorro: 'ja_JP', delon: 'ja_JP', fileName: 'ja-JP.json' },
```

Comment:
The `fileName` value can'n be used, unless provided language pack for [ng-alain](https://github.com/ng-alain/ng-alain/blob/master/src/assets/tmp/i18n/en-US.json) scaffolding.